index.js

```diff
@@ -3,30 +3,58 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+  let i = {};
+  while (i< rolls.length){
```

Comment:
you initiated `i` with an empty object, so this conditional is comparing `{}` to a number, so your while loop will never execute.